src/CMakeLists.txt

```diff
@@ -125,12 +125,17 @@ endif()
 unset(${PROJECT_NAME}_soversion)
 
 if(SECP256K1_BUILD_BENCHMARK)
+  find_library(RT_LIBRARY rt)
+  add_library(optional_rt INTERFACE)
+  if(RT_LIBRARY)
+    target_link_libraries(optional_rt INTERFACE rt)
+  endif()
   add_executable(bench bench.c)
```

Comment:
Note that CMake requires that

* test names are unique *per directory*
* target names are unique.

While it is not strictly required to prefix test names with the project, it is still recommended, because it allows users to control which tests to run: `ctest -R secp256k1_`. This recommendation is already followed, good.

For target names, this requirement is much stronger. Since this project is used as a subproject in bitcoin, neither bitcoin nor any other of its subprojects can define a target with the name `bench`. It would be recommended to rename the target to `secp256k1_bench`.